lib/screens/home_page/home_page.dart

```diff
@@ -84,34 +92,32 @@ class _HomePageState extends State<HomePage> {
                   return Container(
                     width: size.width,
                     height: size.height * 0.76,
-                    child: SingleChildScrollView(
-                      child: ListView.builder(
-                        shrinkWrap: true,
-                        itemCount: snapshot.data!.length,
-                        itemBuilder: (context, int index) {
-                          return ListTile(
-                            leading: Icon(
-                              Icons.book,
-                              size: responsive.dp(3),
-                            ),
-                            title: Text(
-                              snapshot.data![index].title,
-                              style: TextStyle(
-                                fontSize: responsive.dp(1.8),
-                              ),
+                    child: ListView.builder(
+                      shrinkWrap: true,
+                      itemCount: snapshot.data!.length,
+                      itemBuilder: (context, int index) {
```

Comment:
Add here the type. I know the type is inferred but it's better to put it explicitly: `(BuildContext context, int index)`